service/store/v1/store.go

```diff
@@ -169,14 +163,36 @@ func (s *Store) removeExistingRSAKeys() error {
 	return createFolder(s.keyFolderPath)
 }
 
+func readFile(fileName string) ([]byte, error) {
+	content, err := ioutil.ReadFile(fileName)
+	if err != nil {
+		return nil, err
```

Comment:
As of go 1.16 ioutil should be avoided, and features from this package should be replaced by the io and os pacakge.

What about introducing that in here while doing the changes on files restriction? could be a good time to migrate to the new fs.FS api too: https://pkg.go.dev/io/fs#FS.

What do you thing?

Reply:
You are right